python/pyspark/sql/udf.py

```diff
@@ -157,7 +157,17 @@ def _create_judf(self):
         spark = SparkSession.builder.getOrCreate()
         sc = spark.sparkContext
 
-        wrapped_func = _wrap_function(sc, self.func, self.returnType)
+        func = fail_on_stopiteration(self.func)
+
+        # prevent inspect to fail
+        # e.g. inspect.getargspec(sum) raises
+        # TypeError: <built-in function sum> is not a Python function
```

Comment:
Also, I guess the buildin functions already don't work with Pandas UDFs in Python 2 since we already use `getargspec`? i think there's no point of try-except.